DP3TApp.xcodeproj/xcshareddata/xcschemes/Debug.xcscheme

```diff
@@ -64,7 +64,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      language = "de"
```

Comment:
I think the default debugging language should remain german.

Reply:
Fixed.